crates/broker/src/market_monitor.rs

```diff
@@ -295,7 +299,7 @@ where
 
                 while from_block <= to_block {
                     let chunk_end = std::cmp::min(
-                        from_block.saturating_add(lookback_blocks - 1),
+                        from_block.saturating_add(events_poll_blocks.saturating_sub(1)),
```

Comment:
lookback_blocks was what this new `events_poll_blocks` was. If you want to rename it, why not just alias under this new name? Did I misread that the `lookback_blocks` is intended to be used for something else?